### DISCUSSION TITLE
Facial expression

### DISCUSSION THREAD

**Discussion by AyoAyomide** [srcnalt/3d-profile-avatars#5] [Ideas]
This is nice, would be cool if there is a way to integrate Facial expression eg:
model.face.smile();
model.face.wink();
model.face.angry();

**Comment by srcnalt** [comment]
Hi, thank you for the suggestion. I have plans to add such emotions the models support many blend shapes already all there is to do is to implement it.

**Reply by AyoAyomide** [reply]
okay, cool can't wait.. still checkout wolf and readyplayerme docs

**Reply by srcnalt** [reply]
Please do so, you can still do so much more with the avatars.
